libminifi/src/utils/StringUtils.cpp

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
-1 on this change. Had issues linking on arm devices with changes like these. It's like a cmake issue, but let's not make any structural changes, especially in PRs.

Reply:
Actually, this looks a little different and may not have the same linking issues but I'm -1 on this in this PR. Can you submit this into a separate PR so it can be prioritized whether we should include this into this release?

Reply:
A cpp file was created because constexpr static members were added to the StringUtils class which necessitated it.
Because it was created I made the trivial change of moving the functions I could to the cpp file, but that is not necessary.

Reply:
Ah yeah, totally makes sense, thanks for explaining that!